### ISSUE TITLE
Configure shared dir?

### ISSUE THREAD

**Issue by ryanto** [tedconf/ember-cli-front-end-builds#4]
> ember deploy:assets - Uploads assets. Note, this uses whatever is currently in dist folder.
> this command dumps twice, once to a shared dir and once to a uniq dist dir

Is it possible to configure this shared directory? It's dumping in root.

**Comment by ryanto** [comment]
Ok, I think `prefix` is what I want. I think this should have a sane default, like 'dist' or something.

**Comment by samselikoff** [comment]
yep for sure. Right now it does the shared and dist-specific builds, maybe we only need shared?
